Snakemake/workflow/rules/merge_samples_vcf.smk

```diff
@@ -8,29 +8,30 @@ if len(allFiles) != 1:
         output: temp(f'{vcfdir}/{{chromosome}}/{{chromosome}}_{{file}}.txt')
         conda: "bcftools"
         threads: 1
-        resources: cpus=1, mem_mb=4000, time_min=5
-        log: 'logs/get_samples_{chromosome}_{file}.log'
+        resources: cpus=1, mem_mb=32000, time_min=30
+        log: 'logs/Get_samples_{chromosome}_{file}.log'
         shell:
```

Comment:
I've changed everything to be capiralized just to have the same editing everywhere :)